lib/oneview-sdk/resource/api1200/c7000/enclosure_group.rb

```diff
@@ -23,6 +23,10 @@ class EnclosureGroup < OneviewSDK::API1000::C7000::EnclosureGroup
         def initialize(client, params = {}, api_ver = nil)
           @data ||= {}
           # Default values:
+          super
+        end
+
+        def update(attributes = {})
           @data['type'] ||= 'EnclosureGroupV8'
```

Comment:
not sure why there is a need to pass type in the update method as Type attribute should be coming from retrieve resource.